CondCore/BeamSpotPlugins/interface/BeamSpotPayloadInspectorHelper.h

```diff
@@ -632,4 +632,330 @@ namespace BeamSpotPI {
+      f_payload = this->fetchPayload(std::get<1>(firstiov));
+
+      std::string lastIOVsince = std::to_string(std::get<0>(lastiov));
+      std::string firstIOVsince = std::to_string(std::get<0>(firstiov));
+
+      TCanvas canvas(
+          "Sim Beam Spot Parameters Difference Summary", "Sim Beam Spot Parameters Difference summary", 1000, 1000);
+      canvas.cd(1);
+      canvas.cd(1)->SetTopMargin(0.10);
+      canvas.cd(1)->SetBottomMargin(0.06);
+      canvas.cd(1)->SetLeftMargin(0.23);
+      canvas.cd(1)->SetRightMargin(0.16);
+      canvas.cd(1)->Modified();
+      canvas.cd(1)->SetGrid();
+
+      // for the "text"-filled histogram
+      auto h2_SimBSParameters = std::make_unique<TH2F>("Parameters", "", 1, 0.0, 1.0, 9, 0, 9.);
+      h2_SimBSParameters->SetStats(false);
+      h2_SimBSParameters->GetXaxis()->SetBinLabel(1, "Value");
+      h2_SimBSParameters->GetXaxis()->LabelsOption("h");
+      h2_SimBSParameters->GetYaxis()->SetLabelSize(0.05);
+      h2_SimBSParameters->GetXaxis()->SetLabelSize(0.05);
+      h2_SimBSParameters->SetMarkerSize(1.5);
+
+      // prepare the arrays to fill the histogram
+      SimBeamSpotPI::SimBSParamsHelper fBS(f_payload);
+      SimBeamSpotPI::SimBSParamsHelper lBS(l_payload);
+
+#ifdef MM_DEBUG
+      std::stringstream ss1, ss2;
+      edm::LogPrint("") << "**** first payload";
+      fBS.printDebug(ss1);
+      edm::LogPrint("") << ss1.str();
+      edm::LogPrint("") << "**** last payload";
+      lBS.printDebug(ss2);
+      edm::LogPrint("") << ss2.str();
+#endif
+
+      const auto diffPars = fBS.diffCentralValues(lBS);
+      //const auto pullPars = fBS.diffCentralValues(lBS,true /*normalize*/);
+
+      unsigned int yBin = 9;
+      for (int foo = parameters::X; foo <= parameters::timeOffset; foo++) {
+        parameters param = static_cast<parameters>(foo);
+        std::string theLabel = SimBeamSpotPI::getStringFromParamEnum(param, true /*use units*/);
+        h2_SimBSParameters->GetYaxis()->SetBinLabel(yBin, theLabel.c_str());
+        h2_SimBSParameters->SetBinContent(1, yBin, diffPars[foo]); /* profiting of the parameters enum indexing */
+        yBin--;
+      }
+
+      // for the "colz"-filled histogram (clonde from the text-based one)
+      auto h2_SimBSShadow = (TH2F*)(h2_SimBSParameters->Clone("shadow"));
+      h2_SimBSShadow->GetZaxis()->SetTitle("#Delta Parameter(payload A - payload B)");
+      h2_SimBSShadow->GetZaxis()->CenterTitle();
+      h2_SimBSShadow->GetZaxis()->SetTitleOffset(1.5);
+
+      // this is the fine gradient palette (blue to red)
+      double max = h2_SimBSShadow->GetMaximum();
+      double min = h2_SimBSShadow->GetMinimum();
+      double val_white = 0.;
+      double per_white = (max != min) ? ((val_white - min) / (max - min)) : 0.5;
+
+      const int Number = 3;
+      double Red[Number] = {0., 1., 1.};
+      double Green[Number] = {0., 1., 0.};
+      double Blue[Number] = {1., 1., 0.};
+      double Stops[Number] = {0., per_white, 1.};
```

Comment:
I thin these should be all lowercase

Reply:
Addessed in [26deb85](https://github.com/cms-sw/cmssw/pull/41918/commits/26deb853727d29dfe01aa066a676afc14b8001a0)